docs/welcome.mdx

```diff
@@ -4,18 +4,26 @@ title: Welcome to Ory!
 sidebar_label: Welcome
 ---
 
-Ory is an open source ecosystem and a cloud offering authentication, authorization, access control, and delegation (OAuth2 &
-OpenID Connect) services and APIs:
-
-- [Ory Cloud](https://console.ory.sh) is a cloud service built on top of Ory Open Source.
-- Ory Open Source offers scalable and hardened security software:
-- [Ory Kratos](https://github.com/ory/kratos)
-- [Ory Hydra](https://github.com/ory/hydra)
-- [Ory Oathkeeper](https://github.com/ory/oathkeeper)
-- [Ory Keto](https://github.com/ory/keto)
-
-Some parts of Ory are actively being developed. To learn more about roadmap and planned features head over to
-[the roadmap](./ecosystem/roadmap.md) and [project maturity](./ecosystem/versioning.md).
+Ory is a software infrastructure provider building a global zero trust network for humans, robots, devices, and logical services.
+Ory fuses open source software (Ory, Kubernetes, Crossplane, Cockroach, Linux, Istio) and open standards (Ory, OAuth2, OpenID
+Connect, MITREid, WebAuthn, TOTP) with centuries in combined open source, security, operational, and industry expertise.
+
+[Ory Cloud](https://console.ory.sh)'s core infrastructure components are open-source to foster collaboration, reduce security
+supply-chain risk, make security accessible, and introduce the open standard for internet security. Being open source, Ory
+improves the security of everyone in an age of global volatility:
+
+- The Ory Identity Service is developed as an open source project called [Ory Kratos](https://github.com/ory/kratos), introducing
+  a new open standard for identity management.
+- The Ory OAuth2 & OpenID Connect Service is developed as an open source project called [Ory Hydra](https://github.com/ory/hydra),
+  implementing 15+ IETF and OpenID standards.
+- The Ory Permission Service is developed as an open source project called [Ory Keto](https://github.com/ory/keto), implementing
+  _[Zanzibar: Google’s Consistent, Global Authorization System](https://research.google/pubs/pub48190/)_ by Google.
+
+Ory develops and maintains many additional open source projects, such as the Ory Zero Trust Identity & Access Proxy
+[Ory Oathkeeper](https://github.com/ory/oathkeeper), developer tooling such as [Ory Dockertest](http://github.com/ory/dockertest)
+and language-specific libraries such as [Ory Ladon](https://github.com/ory/ladon).
```

Comment:
I think presenting this information as a bulleted list would be nice.

Reply:
Yeah I thought about that, but I did indeed wanted it to not use so much space. The rest of the document also needs to be refactored. It should basically be 20% of screen dedicated to quickly explaining what Ory is, and the rest of it to where to go next.